### DISCUSSION TITLE
List of all tags in tag-field

### DISCUSSION THREAD

**Discussion by vluijkx** [laravel/nova-issues#5077] [Ideas & Feature Requests]
Hi,

We have lot's of use-cases for the new Tag field. In most of these cases it would be really helpful to see a list of all available tags while editing. Especially for people not using the admin-panel frequently, it is hard to guess the right term. We are often finding ourselves thinking things like "What did we name this role again?". There are also many synonyms created as people did not come across an existing perfectly fitting entity.

I was wondering how you guys think about this.

**Comment by sutcliffe121** [comment]
I also have the same use case. It would be useful for users to be able to see a list of available tags, as well as search.

After creating a new tag inline, a list of tags appears - so the functionality is there. It would be useful to have this available all the time.

![image](https://user-images.githubusercontent.com/33236577/203164170-efe08db4-0f87-4bf2-99d4-8f4028bc87ff.png)

**Comment by sutcliffe121** [comment]
Are there any comments/updates on this?

If you create a new tag inline and subsequently click into the tag field... all the available tags are displayed and available to select.

This is the required behaviour before creating a new tag inline.

**Comment by ziming** [comment]
This is duplicate of an older discussion thread.

https://github.com/laravel/nova-issues/discussions/5011

So 10 upvotes in total at the time of this post. Maybe can close 1 of the 2 to keep discussions in 1 place.

**Comment by ghost** [comment]
Yes, please, this is really needed. Either by adding ->searchable() method to enable/disable current behavior or ->options() for passing initial list of values.

**Comment by ghost** [comment]
Hi @davidhemphill. Is this on your radar?

**Comment by sutcliffe121** [comment]
Are there any plans to address this issue @davidhemphill ? It would make the Tag field much more useable.

**Comment by orlyapps** [comment]
this would be very helpful :) 
Thank you for all the work ♥️

**Comment by davidhemphill** [comment]
This has been added and will be included in the next release. 👍
